superdesk/publish/formatters/ninjs_formatter.py

```diff
@@ -289,8 +289,6 @@ def _format_related(self, article, subscriber):
         for key, item in (article.get(ASSOCIATIONS) or {}).items():
             if item:
                 associations[key] = self._transform_to_ninjs(item, subscriber)
-            else:
-                associations[key] = None
```

Comment:
This will generate a problem on updating the content api after an item from associations is deleted.

Reply:
that's why I've added this test - https://github.com/superdesk/superdesk-core/pull/1164/files#diff-57be8e755f04f5d58c1694174ac4baafR200 - doesn't seem to be a problem, because that code which merges it with original is only applied when doing patch request, while the content api is writing it directly to the storage